### DISCUSSION TITLE
Automatic removal on completion

### DISCUSSION THREAD

**Discussion by umop3plsdn** [sdaqo/anipy-cli#211] [Ideas]
Whenever i run anipy-cli -H  its clogged up with anime's i've finished. I mean I could delete the entire history but its a pain when i have a bunch of others that i'm still watching. I wouldn't mind a way to just delete one of them. but I think the thing that makes the most sense is if its on the last episode to just remove it from the history altogether no?

**Comment by sdaqo** [comment]
Well it's just... the history, and I think all the history should be in the history, if that makes sense.

I presume you use the history to know which episode of which anime you last watched? Maybe you can try using seasonal mode, you can track your anime with it. Maybe have a look at this thread: https://github.com/sdaqo/anipy-cli/discussions/202

Edit: specifically, I mean [this](https://github.com/sdaqo/anipy-cli/discussions/202#discussioncomment-11009954) comment

**Comment by sdaqo** [comment]
Maybe I should rename it and rework it a little bit to be a general "tracker" mode instead of just a seasonal mode. 🤔
